styles/style.css

```diff
@@ -6,3 +7,180 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  
+    font-family: Arial, Helvetica, sans-serif;
+    /* Is there a reason why the font circular doesnt populate?*/
+    color: #1A1A1A;
+    font-size: 16px;
+    
+}
+
+img {
+    width: 300px;
+}
+
+
+
+nav {
+    height: ;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 1rem;
+    height: 75px;
+}
+
+
+.navlist {
+    list-style-type: none;
+    display: flex;
+}
+
+.navlogo {
+    width: 10rem;
+}
+
+.navlist > li {
+    padding: 0 1rem;
+    font-size: 1.1rem;
+
+}
+
+
+header {
+    background-image: url("../images/landing.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 500px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+	align-items: center;
+}
+
+h1 {
+    margin-top: -75px;
```

Comment:
Negative margins can be handy, but keep in mind that it can also screw up the code as well since it will put the element on top of whatever comes above it. I would in this case have set the margin: 0, however using it like you do here is perfectly fine:)